modules/installation-azure-permissions.adoc

```diff
@@ -6,10 +6,11 @@
 [id="installation-azure-permissions_{context}"]
 = Required Azure roles
 
-Your Microsoft Azure account must have the following roles for the subscription
+You can use your Microsoft Azure account for `az login` and browsing in the web console. The Microsoft Azure account has the correct permissions to create the service principal account. The created service principal account is used to run the installer and create the cluster. Your Microsoft Azure account must have the following roles for the subscription
 that you use:
 
 * `User Access Administrator`
+* `Owner`
```

Comment:
How did we determine that the Owner role is required? Is the User Access Administrator role alone not sufficient?  We should only add the bare minimum amount of required roles.

Reply:
>  Is the User Access Administrator role alone not sufficient?
Right. QE tried that. https://bugzilla.redhat.com/show_bug.cgi?id=2025482#c9

Reply:
Ok this makes sense. Thank you.